pilot/lib/pilot/tester_manager.rb

```diff
@@ -6,19 +6,27 @@ module Pilot
   class TesterManager < Manager
     def add_tester(options)
       start(options)
-
       app = find_app(app_filter: config[:apple_id] || config[:app_identifier])
       UI.user_error!("You must provide either a Apple ID for the app (with the `:apple_id` option) or app identifier (with the `:app_identifier` option)") unless app
 
-      tester = find_or_create_tester(email: config[:email], first_name: config[:first_name], last_name: config[:last_name])
-
+      tester = find_app_tester(email: config[:email], app: app)
+      tester ||= create_tester(
+        email: config[:email],
+        first_name: config[:first_name],
+        last_name: config[:last_name],
+        app: app
+      )
       begin
         groups = Spaceship::TestFlight::Group.add_tester_to_groups!(tester: tester, app: app, groups: config[:groups])
         if tester.kind_of?(Spaceship::Tunes::Tester::Internal)
           UI.success("Successfully added tester to app #{app.name}")
         else
-          group_names = groups.map(&:name).join(", ")
-          UI.success("Successfully added tester to app #{app.name} in group(s) #{group_names}")
+          if config[:groups]
+            group_names = groups.map(&:name).join(", ")
+            UI.success("Successfully added tester to group(s): #{group_names} in app: #{app.name}")
+          else
+            UI.success("Successfully added tester to the default tester group in app: #{app.name}")
```

Comment:
Where is that being done? Can you add a comment where it explains where that is?